app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,30 @@
+
+<table>
+  <tr>
+    <th>id</th>
+    <th>name</th>
+    <th>master of the task</th>
+  </tr>
+
+<% task_num = 0 %>
+<% @tasks.each do |t|%>
+
+  <tr class="<%= t.completed ? 'completed' : 'uncompleted' %>" >
+      <% @id = t.id%>
```

Comment:
You don't need to use an instance variable because this information is just needed within this view. This should be a local variable.